crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -560,7 +560,7 @@ def proposal_update
       #
 
       if params[:submit] == t("barclamp.proposal_show.save_proposal")
-        @proposal = Proposal.where(barclamp: params[:barclamp], name: params[:id] || params[:name]).first
+        @proposal = Proposal.find_by(barclamp: params[:barclamp], name: params[:id] || params[:name])
```

Comment:
Metrics/LineLength: Line is too long. [101/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)